pkg/actions/gvr.go

```diff
@@ -34,3 +37,12 @@ func GetGroupVersionResource(gr schema.GroupVersionResource, discovery discovery
 
 	return &gvr, nil
 }
+
+func InitializeAPIGroupRes(discovery discovery.DiscoveryInterface) error {
+	var err error
+	apiGroupRes, err = restmapper.GetAPIGroupResources(discovery)
```

Comment:
```suggestion
	apiGroupRes, err := restmapper.GetAPIGroupResources(discovery)
```

Reply:
as `apiGroupRes` has been declared before, this is throwing errors